clang/test/dpct/helper_files_ref/include/math.hpp

```diff
@@ -33,6 +33,9 @@ class vectorized_binary<
     return binary_op(a, b).template as<VecT>();
   }
 };
+
+template <typename T> bool isnan(const T a) { return sycl::isnan(a); }
+// TODO: Need add more specialization.
```

Comment:
be more clear why need more specialization

Reply:
fixed